src/main/java/racinggame/domain/Cars.java

```diff
@@ -0,0 +1,44 @@
+package racinggame.domain;
+
+import racinggame.domain.rule.RandomMoveStrategy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private final List<Car> carList;
```

Comment:
```suggestion
    private final List<Car> carList = new ArrayList<>();
```

자료구조타입의 변수들은 필드 초기화를 해주면 npe에서 좀 더 자유로워집니다 ㅎㅎ

Reply:
private Cars(List<Car> cars) {
        this.carList = new ArrayList<>(cars);
    }
  
생성자 부분에 초기화를 해주는건 npe를 방지 하지 못하는건가요?

Reply:
항상 저 생성자로만 객체를 생성하고 그게 바뀌지 않는다면, 문제가 없겠지만, 차후 다른 추가 생성자가 생기거나 할 경우에 초기화 로직을 빼먹으면 npe 이슈가 생길 수 있으니까요 ㅎㅎ